TrainingMaquetado/scss/header.scss

```
@@ -0,0 +1,18 @@
+.header {
+    align-items: center;
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 10px 0px 0px 20px;
+
+    .header-title{
+      color: black;
+      font-size: 24px;
+      margin-left: -18px;
+    }
+
+    .header-logo{
```

Same. Also leave a white space between your class name and the bracket 

```suggestion
    .header-logo {
```